apps/frontend/src/components/global/ExportSplunkModal.vue

```diff
@@ -38,26 +38,40 @@
             absolute="absolute"
             :value="errorCount >= 3 || errorCount < 0"
           >
-            <div class="text-center">
+            <div class="text-left">
               <p>
                 <span v-if="errorCount > 0">
                   It seems you may be having trouble using the Splunk toolkit.
                   Are you sure that you have configured it properly?
                 </span>
                 <br />
                 <span>
-                  For installation instructions and further information, check
-                  here:
+                  Accessing a Splunk instance from Heimdall requires the input
+                  of the following information:
+                  <br />
+                  username: A qualified username recognized by the referenced
+                  Splunk instance.
+                  <br />
+                  password: A qualified password recognized by the referenced
+                  Splunk instance.
+                  <br />
+                  hostname: The domain name for the desired Splunk instance.
+                  Include port number if available.
+                  <br />
+                  index: A valid index name within the referenced Splunk
+                  instance.
+                  <br />
+                  For installation instructions and further information, see:
                 </span>
                 <v-btn
                   target="_blank"
-                  href="https://github.com/mitre/hdf-json-to-splunk/"
```

Comment:
can you validate that this plugin is basically just the splunk-hdf mapper but over in splunk instead?  doing a brief skim, it seems to have other functionality embedded in there as well like some sorta script to translate files from s3 into splunk.  we should ask in standup if we know if anyone is still using this plugin cause currently it has been unmaintained for 3-4 years, and i'd like to officially archive it if possible.  i didn't notice any dependencies aside from the sdk being like hard embedded in the repo for some reason so hopefully there's no security concerns if we are in fact forced to leave it up.  finally, if we are able to archive it, we'll probably be breaking some links since we have a custom dns entry up which we should probably have redirect to some landing page on the saf site saying that whatever you tried accessing has been discontinued.  the reason we have to do that is if we archive it without doing anything about the domain name existing, then folks can grab it and point it elsewhere which is kinda wild that that's how the internet works lol.